extensions/blocks/calendly/view.scss

```diff
@@ -10,3 +10,7 @@
 .wp-block-jetpack-calendly .calendly-spinner {
 	top: 50px;
 }
+
+.wp-block-jetpack-calendly.aligncenter .wp-block-jetpack-button {
+	text-align: center;
```

Comment:
I think we should also center it in the editor.

Reply:
@copons Blocks with the button center-aligned are showing fine for me in the editor due to to the following CSS:

```
.block-editor-block-list__block[data-align=center] {
     text-align: center;
}
```

Are you aware of a case where this is insufficient?

Reply:
Apologies, I should have added more context.
I've tested (and reproduced again today) with the Gutenberg plugin active (v8.3.0).

I can also confirm that the button centers correctly when the plugin is deactivated.

Reply:
Apparently the block alignment behaviour changed slightly.
Instead of applying the `data-align` to the block div, it now creates a wrapper `.wp-block` div with the `data-align`.

Reply:
Thanks for pointing that one out. I've tweaked the styles and its working for me with the Gutenberg plugin (8.3.0) installed and active as well as without it.